examples/resources/hpegl_vmaas_load_balancer_profile/nsx_t_lb_profile.tf

```diff
@@ -1,6 +1,7 @@
 # (C) Copyright 2022 Hewlett Packard Enterprise Development LP
 
 resource "hpegl_vmaas_load_balancer_profile" "tf_lb_profile" {
+  lb_id = data.hpegl_vmaas_load_balancer.lb.id  
   name  =  "LB-PROFILE"       
   description  = "creating LB Profile"
   service_type     = "LBHttpProfile"
```

Comment:
based on profile_type, the below fields changes for the user. It is highly diffficult for the end user to understand a clubbed resource definition to configure a given type of profile type. My suggestion would be separate it out for each profile. Currently for HTTP Service Type, there are more fields which are displayed in the UI, but not used in IaC.

The best way going forward is to analyse the resource definition before implementing it and see how can it be made helpful for the user consumption. 

Please refer Tier0/Tier1 Router creation for your reference.

I would suggest to take each of these Pools, Profiles and Monitors and Servers as independent stories so that we implement them in a right way.

Please let us know if we need a meeting to understand this better.